app/models/time_profile.rb

```diff
@@ -75,7 +73,7 @@ def tz_or_default(default_tz = DEFAULT_TZ)
   end
 
   def days
-    profile[:days]
+    profile.fetch(:days, ALL_DAYS)
```

Comment:
I believe I tried that and existing tests failed because it's expecting this hash structure on the getter since it's done on the setter.  I 100% agree with the idea of migrating this out and simplifying it.  I don't know why it was done this way.

> the default_value overriding what comes back from the database (vs creating the object) is a little suspect. Glad we are getting rid of this.

Good point.  Let me check into it.  most of the defaults were to set it if it's not set.  We probably don't have good tests if I was able to ignore the value in the database.  I think you found a bug.

Reply:
Ok, I took a look.  I think this is fine for now.  I agree, we should stop storing the days/hours in the profile column.